crimsobot/utils/tarot.py

```diff
@@ -1,5 +1,3 @@
-import io
-import random
```

Comment:
wait why were these removed

Reply:
Oh, whoops. Was removing an accidental newline, must have selected too far

Reply:
Should be clean now.